scitt_emulator/scitt.py

```diff
@@ -168,9 +169,23 @@ def _sync_policy_result(self, operation: dict):
             policy_insert_path.unlink()
         if policy_failed_path.exists():
             policy_result["status"] = "failed"
+            with contextlib.suppress(
+                ValueError,
+                TypeError,
+                RecursionError,
+                json.JSONDecodeError,
+            ):
+                policy_result.update(json.loads(policy_failed_path.read_text()))
             policy_failed_path.unlink()
         if policy_denied_path.exists():
             policy_result["status"] = "denied"
+            with contextlib.suppress(
+                ValueError,
+                TypeError,
+                RecursionError,
+                json.JSONDecodeError,
+            ):
+                policy_result.update(json.loads(policy_denied_path.read_text()))
```

Comment:
I don't currently see logging infrastructure but we should add some here if we add logging code.